lib/builder.js

```diff
@@ -61,6 +62,7 @@ const configSchema = {
                 rebalanceTimeout: { type: 'integer', default: 60000 },
                 heartbeatInterval: { type: 'integer', default: 3000 },
                 allowAutoTopicCreation: { type: 'boolean', default: false },
+                maxWaitTimeInMs: { type: 'integer', default: DEFAULT_MAX_WAITING_TIME_MS, description: 'The maximum amount of time in milliseconds the server will block before answering the fetch request if there isn’t sufficient data to immediately satisfy the requirement given by minBytes' },
```

Comment:
What do you think about the addition of a generic `additionalProperties` config that the user can pass as parameter? In this way we don't need to explicitly add the kafkajs properties that we need and the developers are free to pass every property to kafka

Reply:
That could do, so that no other change would be required to add further config properties to consumer. We should add `additionalProperties: true` also in the producer config.

Nonetheless, I would prefer to also keep current edit, so that the `maxWaitTimeInMs` value is set to Kafka's default one, instead of the [much larger value](https://github.com/tulios/kafkajs/blob/d67b22eb58e12f6d98f4ae18986dd193019a9018/src/consumer/index.js#L60) set by Kafkajs library.

Reply:
is it really necessary to explicitely add additionalProperties: true, it should  be true if not specified no ?

Reply:
I wouldn't call the option "additionalProperties" if they'll contain kafkajs-related configurations; i'd wrap them in a `kafkajsOptionts` configured with `additionalProperties:true`

Reply:
I agree with @fredmaggiowski, I used a very bad example as property name 😂